src/AWS/Signers/V4.elm

```diff
@@ -143,4 +154,11 @@ stringToSign algorithm date scope canon =
 
 jsonContentType : AWS.Config.Service -> String
 jsonContentType config =
-    "application/x-amz-json-" ++ config.xAmzJsonVersion ++ "; charset=utf-8"
+    (case config.xAmzJsonVersion of
+        Just version ->
+            "application/x-amz-json-" ++ version
+
+        Nothing ->
+            "application/json"
```

Comment:
Some apis do not specify a json version, so we'll just use vanilla `application/json` in that case. Haven't verified this in the AWS docs yet.